ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
+
+
+def meters_to_long(x):
+    # Converts longitude point from Spherical Mercator EPSG:900913 to lat/lon in WGS84 Datum
+    origin_shift = 2 * math.pi * 6378137 / 2.0
+    long = (x / origin_shift) * 180.0
+    return long
+
+
+def meters_to_lat(y):
+    origin_shift = 2 * math.pi * 6378137 / 2.0
+    lat = (y / origin_shift) * 180.0
+    lat = 180 / math.pi * (2 * math.atan(math.exp(lat * math.pi / 180.0)) - math.pi / 2.0)
+    return lat
+
+
+def get_localities(url, username, password):
```

Comment:
If you're planning to pass in passwords, would recommend something that lets you type it in obscured text. I have used `getpass` (https://docs.python.org/3.1/library/getpass.html) in the past -- but that was like 3 years ago...

Reply:
Hmmm, I'll take a look into it! As long as it works with Tkinter apps (and I don't see why it wouldn't!)